lib/slack_message.ml

```diff
@@ -132,49 +132,62 @@ let month = function
   | 12 -> "Dec"
   | _ -> assert false
 
-let populate_commit repository (commit : api_commit) =
+let condense_file_changes files =
+  match files with
+  | [ f ] -> sprintf "_modified `%s` (+%d-%d)_" (escape_mrkdwn f.filename) f.additions f.deletions
```

Comment:
this doesn't look right for single element case - the output of this function gets wrapped in another "modified..." string below.

also maybe naming the function something like `make_files_prefix_path` (not sure) would make it clearer what it returns

Reply:
ah good catch thanks!

Reply:
is it fixed?

Reply:
yep

Reply:
but what if the input is `["a.ml"; "b.ml"]`?

Reply:
The function is used in two places.

For the above example I'd still expect the final text to say "modified 2 files" even though it's at the root. At least, that's the current behavior for single commit unfurl, which this function also affects. With your change, it returns an empty string.

Reply:
Hmm, I see--I can modify that!

Reply:
okay, it's updated @yasunariw !

Reply:
fb01b81 fixes the regression introduced to markup for single commit links. Although, now the tests for compare links have `\n\n\nmodified 14 files` - are so many linebreaks expected?

Reply:
yep-i rm-ed one `\n`, i think 3 is excessive :)